src/jsonLogic.js

```diff
@@ -107,32 +109,106 @@ export function yupSchemaWithCustomJSONLogic({ field, validations, config, id })
     );
 }
 
+function replaceHandlebarsTemplates({
+  value: toReplace,
+  validations,
+  formValues,
+  parentID,
+  name: fieldName,
+}) {
+  if (typeof toReplace === 'string') {
+    return toReplace.replace(/\{\{([^{}]+)\}\}/g, (match, key) => {
+      return validations
+        .getScope(parentID)
+        .evaluateComputedValueRuleForField(key.trim(), formValues, fieldName);
```

Comment:
nit: Following my comments from #34, I'm now more confident that this fn should be called `apply*` or `get*` because we are really returning that value, not just "evaluating" true/false

Reply:
Appears that it's been changed to `applyComputedValueInField`, so resolving this one.